src/components/CareCardPrevention/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './CareCardPrevention.scss';
+import preventionImage from '../../assets/images/prevention.svg';
+
+const CareCardPrevention = () => (
+  <React.Fragment>
```

Comment:
Here the same observation made in the card component, this `<React.Fragment>` can be avoided.